source/whats-new.txt

```diff
@@ -29,6 +30,22 @@ Learn what's new in:
 * :ref:`Version 4.11 <version-4.11>`
 * :ref:`Version 4.10 <version-4.10>`
 
+.. _java-version-5.3:
+
+What's New in 5.3
+-----------------
+
+The 5.3 {+driver-short+} release introduces the following improvements, features,
+and fixes:
+
+- Client bulk write API that allows you to perform write operations on multiple
+  databases and collections at once. To learn more about this feature, see the
+  :ref:`java-sync-client-bulk-write` section of the Bulk Operations guide.
+
+- A ``ClientBulkWriteOptions`` object to specify different options for how
+  the client bulk write API is executed. See the :ref:`java-sync-client-bulk-write-options` 
+  section of the Client Bulk Write guide.
```

Comment:
S: you can leave this for tech review to decide on, but not sure that specifically calling this out is necessary in the whats new

Reply:
i will remove! also wasn't sure it was necessary